app/controllers/admin/users_controller.rb

```diff
@@ -10,11 +10,22 @@ def show; end
 
   def edit; end
 
-  def update
-    if @user.update(user_params)
-      redirect_to admin_users_path
+  def update 
+    if params[:user][:password].blank? && params[:user][:password_confirmation].blank? # if password fields blanks
```

Comment:
what if only one of them is balnk?

Reply:
Also can we use `user_params[:password]` and `user_params[:password_confirmation]`